### ISSUE TITLE
[BUG] Documentation Error for TFT getMatchIdsByPUUID

### ISSUE THREAD

**Issue by Protroid** [RiotGames/developer-relations#214]
Hello!

Looking through the documentation, I noticed that the puuid path parameter is marked as optional on the TFT getMatchIdsByPUUID API call. Leaving this value blank results in a 400 error, with a message of: "Bad Request - Exception decrypting %7Bpuuid%7D".

If it is intended to be optional, the logic may need to be looked over, otherwise the documentation needs to be updated to be marked as required.

Thanks!

**Comment by MingweiSamuel** [comment]
I agree it doesn't make sense to be optional since it is a path param

**Comment by Querijn** [comment]
This breaks the swagger generator I use, breaking my path generation.
